zio/detector/lookup.go

```diff
@@ -22,11 +23,18 @@ func (*nullWriter) Write(*zng.Record) error {
 	return nil
 }
 
-func LookupWriter(format string, w io.WriteCloser, optionalFlags *zio.Flags) *zio.Writer {
-	var flags zio.Flags
+func LookupWriter(w io.WriteCloser, optionalFlags *zio.WriterFlags) *zio.Writer {
+	if optionalFlags == nil {
+		panic("FLAGS SHOULDN'T BE OPTIONAL")
+	}
```

Comment:
```suggestion
func LookupWriter(w io.WriteCloser, wrFlags *zio.WriterFlags) *zio.Writer {
	if inFlags == nil {
		panic("required wrFlags not set")
	}
```

and adjust below accordingly

Reply:
Oops, I wasn't done.  I was looking for calls with nil and will just take out the panic and require the flags arg.